src/main/java/xbot/common/controls/actuators/XCANMotorController.java

```diff
@@ -77,13 +77,15 @@ default XCANMotorController create(
     protected boolean usesPropertySystem = true;
     protected boolean firstPeriodicCall = true;
 
-    private DoubleProperty kPProp;
-    private DoubleProperty kIProp;
-    private DoubleProperty kDProp;
-    private DoubleProperty kVelocityFFProp;
-    private DoubleProperty kGravityFFProp;
-    private DoubleProperty kMaxOutputProp;
-    private DoubleProperty kMinOutputProp;
+    protected DoubleProperty kPProp;
+    protected DoubleProperty kIProp;
+    protected DoubleProperty kDProp;
+    protected DoubleProperty kVelocityFFProp;
+    protected DoubleProperty kGravityFFProp;
+    protected DoubleProperty kMaxOutputProp;
```

Comment:
What do you think about having just one set of these (e.g. just voltage, or just power) and setting both at runtime (e.g. power*12 to volts, or volts/12 to power?) Do you think we have enough cases where we need to keep these separate, weighted against the risk that somebody sets one set of properties and forgets about the other?

Reply:
I'll use one property for the pid settings. Using 13.8 (max charge voltage) as the multiplier to ensure we don't accidentally lose extra voltage on a fresh battery.

setVoltage and setPower remain separate, I expect those only to be used when needed for some special case anyways.